src/scandeval/utils.py

```diff
@@ -21,6 +21,21 @@
 from .exceptions import InvalidBenchmark
 
 
+def create_model_cache_dir(cache_dir: str, model_id: str) -> str:
+    """Create cache directory for a model.
+
+    Args:
+        cache_dir: The cache directory.
+        model_id: The model ID.
```

Comment:
Intentionally didn't add typing here to avoid documenting the typing twice. This also avoids cases where edits in the future can lead to disagreements on the type. I can add it in if you wish.

Reply:
Just had a re-read of Google's formatting guidelines, and you're absolutely right - types are only required in doc strings if the information isn't included as type hints. All good, and maybe I'll do a cleanup of the rest of the code base in the near future 🙂

Reply:
It is also a lot easier to maintain 👌